plugins/services/src/js/components/modals/CreateServiceModalForm.js

```diff
@@ -309,7 +319,18 @@ class CreateServiceModalForm extends Component {
       baseConfigCopy
     );
 
-    return CreateServiceModalFormUtil.stripEmptyProperties(newConfig);
+    const { labels, ...config } = newConfig;
+
+    if (Object.keys(labels).length === 0) {
+      return {
```

Comment:
shouldnt be `return CreateServiceModalFormUtil.stripEmptyProperties(config)` be enough here?